econml/tests/test_ortho_learner.py

```diff
@@ -1,484 +1,487 @@
-# Copyright (c) PyWhy contributors. All rights reserved.
```

Comment:
Any idea why GitHub thinks every line has changed?

Reply:
Seems that the line endings got switched to crlf from lf at some point. Will try and switch back